test/user.js

```diff
@@ -0,0 +1,103 @@
+//During the test the env variable is set to test
+process.env.NODE_ENV = 'development';
+
+let chai = require('chai'),
+  chaiHttp = require('chai-http'),
+  server = require('../app'),
+  should = chai.should(),
+  dictum = require('dictum.js'),
+  User = require('../app/models').users;
+
+chai.use(chaiHttp);
+
+describe('User', () => {
+  beforeEach((done) => {
+    User.destroy({where: {}}, (err) => {}).then(done());     
+  });
+
+  /*
+  * Testing the /users (POST) route
+  */
+  describe('/POST users', () => {
+
+    let newUser = {
+      name: 'Kevin',
+      lastName: 'Temes',
+      email: 'kevin.temes@wolox.com.ar',
+      password: '12345678'
+    };
+
+    it('it should successfully POST a user with the given inputs', (done) => {
+      chai.request(server)
+        .post('/users')
+        .send(newUser)
+        .end((err, res) => {
+          if(err) done(err);
+          res.should.have.status(201);
+          res.body.should.include({name: 'Kevin', lastName: 'Temes', email: 'kevin.temes@wolox.com.ar'});
+          dictum.chai(res, 'description for endpoint');
+          done();
+        });
+    });
+
+    it('it should throw an error when attempting to POST a user with an already existing email', (done) => {
+      chai.request(server)
+        .post('/users')
+        .send(newUser)
+        .end((err, res) => {});
+
+      chai.request(server)
```

Comment:
this request should be nested inside the previous one with a .then, otherwise they can run concurrently and fail unexpectedly.